backend/wordAlgorithm/word.py

```diff
@@ -0,0 +1,94 @@
+# Format of xml taken as input:
+#    <Words>
+#        <Word>
+#            <Language>Dutch</Language>
+#            <Word>ja</Word>
+#            <Translation>yes</Translation>
+#            <Category/>
+#            <Word_Difficulty/>
+#            <Extensions/>
+#        </Word>
+#    </Words>
+
+
+words = []
+
+# Word object
+class Word(object):
```

Comment:
_This is debatable_ Word class i think should contain some functionality. In such classes __str__ function is important to begin with. Also depending on how you use it you might want to add some functions to extend the list:extensions or the int:workDifficulty